### ISSUE TITLE
Incompatible with symfony/yaml ^5

### ISSUE THREAD

**Issue by someniatko** [cebe/php-openapi#56]
Steps to reproduce:
1. require `symfony/yaml ^5`
2. try executing `composer require cebe/php-openapi`

Result:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - cebe/php-openapi 1.3.3 requires symfony/yaml ^3.0 | ^4.0 -> no matching package found.
    - cebe/php-openapi 1.3.2 requires symfony/yaml ^3.0 | ^4.0 -> no matching package found.
    - cebe/php-openapi 1.3.1 requires symfony/yaml ^3.0 | ^4.0 -> no matching package found.
    - cebe/php-openapi 1.3.0 requires symfony/yaml ^3.0 | ^4.0 -> no matching package found.
    - Installation request for cebe/php-openapi ^1.3 -> satisfiable by cebe/php-openapi[1.3.0, 1.3.1, 1.3.2, 1.3.3]
```

**Comment by someniatko** [comment]
related PR: https://github.com/cebe/php-openapi/pull/45

**Comment by cebe** [comment]
Not sure how to solve this, our tests are breaking with symfony/yaml 4.4 and 5.0, but this is due to some specific yaml files that we fail to parse because of the symfony bug https://github.com/symfony/symfony/issues/34805 it does not look like it is going to be fixed soon so I'd like to add 5.0 to the supported symfony versions. For this we need to add some switches to the tests that exclude the failing files from tests with these symfony versions....
